src/category_theory/epi_mono.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2019 Reid Barton. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Reid Barton
+
+Facts about epimorphisms and monomorphisms.
+
+The definitions of `epi` and `mono` are in `category_theory.category`,
+since they are used by some lemmas for `iso`, which is used everywhere.
+-/
+
+import category_theory.adjunction
+import category_theory.fully_faithful
+
+universes v₁ v₂ u₁ u₂
+
+namespace category_theory
+
+variables {C : Sort u₁} [𝒞 : category.{v₁} C] {D : Sort u₂} [𝒟 : category.{v₂} D]
+include 𝒞 𝒟
+
+lemma left_adjoint_preserves_epi {F : C ⥤ D} {G : D ⥤ C} (adj : adjunction F G)
+  {X Y : C} {f : X ⟶ Y} (hf : epi f) : epi (F.map f) :=
```

Comment:
After merging/rebasing onto master, you can use the notation `F ⊣ G` for `adjunction` (also below).

Reply:
Done.